sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/tests/prerequisites/setup.ps1

```diff
@@ -114,7 +114,6 @@ $timeSeriesIdProperties = $timeSeriesIdProperties.Replace('"','\"')
 az deployment group create --resource-group $ResourceGroup --name $($EnvironmentName.ToLower()) --template-file $armTemplateFile --parameters `
     region=$Region `
     resourceGroup=$ResourceGroup `
-    subscriptionId=$SubscriptionId `
```

Comment:
Our ADT setup script still has this parameter and it works. Why must it be removed from here?

Reply:
It does as a parameter for setup script itself. But we don't pass it as a parameter to the ARM template:
https://github.com/Azure/azure-sdk-for-net/blob/a3bd0beb8baead281e3929b15b4b3d3ed6873e94/sdk/digitaltwins/Azure.DigitalTwins.Core/tests/prerequisites/setup.ps1#L97

So in here, I am removing subscription ID as a parameter to the ARM template.

Reply:
Oh, my bad. I see now.